setup/src/main.ts

```diff
@@ -1,20 +1,44 @@
 import * as core from '@actions/core';
 import * as exec from '@actions/exec';
+import * as github from '@actions/github';
+import axios from 'axios';
 
+const SPRKL_RECIPES = ['auto', 'uncommitted', 'mine', 'recent', 'all', 'lastPush', 'commitsList'];
```

Comment:
Cannot be `commitsList`  since the user of the action cannot define the list of commits. 
Therefore we either validate and prompt that it is not supported, or open a window to insert a commits list.

Reply:
For now there is validation that the input recipe isn't commitsList but maybe we should give the user the option to insert a commits list

Reply:
@erez-sprkl what do you think about that?